.cmake.version

```diff
@@ -0,0 +1 @@
+3.26.3
```

Comment:
The added file is just there to ease getting a proper PR title / commit message.
For the commit message, bump -- commit could be used, requiring to setup the bot user before calling bump.

Reply:
This is in pyproject.toml now